src/common/http/src/debug/async_backtrace.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use async_backtrace;
+use poem::IntoResponse;
+
+// run async-backtrace
+// example: /debug/async_backtrace
+// req query contains async_backtrace information
+
+#[poem::handler]
+pub async fn debug_async_backtrace_handler() -> impl IntoResponse {
+    async_backtrace::taskdump_tree(true)
```

Comment:
To reduce the cost of taskdumping, use:
```suggestion
    async_backtrace::taskdump_tree(false)
```

By doing so, `taskdump_tree` won't wait for currently-running tasks to suspend so it can print their state (it'll just skip them).